sentry_sdk/integrations/pymongo.py

```diff
@@ -23,6 +23,52 @@
     )
 
 
+SAFE_COMMAND_ATTRIBUTES = [
+    "insert",
+    "ordered",
+    "find",
+    "limit",
+    "singleBatch",
+    "aggregate",
+    "createIndexes",
+    "indexes",
+    "delete",
+    "findAndModify",
+    "renameCollection",
+    "to",
+    "drop",
+]
+
+
+def _strip_pii(command):
+    # type: (Dict[str, Any]) -> Dict[str, Any]
+    for idx, key in enumerate(command):
+        if key in SAFE_COMMAND_ATTRIBUTES or (key == "update" and idx == 0):
+            # Skip if safe key, or the is "update" but not on the first place
+            # "update" as the first key is safe because it is the mongo db command.
+            # "update" as a later key (for ex in the findAndModify command) is not save and should be stripped of PII.
```

Comment:
This approach won't work in Python < 3.7 where dict ordering became standard. You'd need to explicitly check for command to be `update`.